src/plugins/intel_gpu/src/kernel_selector/kernels/range/range_kernel_ref.cpp

```diff
@@ -13,7 +13,7 @@ namespace {
 CommonDispatchData SetDefault(const range_params &params) {
     CommonDispatchData dispatchData;
 
-    dispatchData.gws = { 1, 1, params.outputs[0].X().v }; // TODO: these could be split better
+    dispatchData.gws = { 1, 1, params.outputs[0].Batch().v }; // TODO: these could be split better
```

Comment:
Is this change applied for the original shape infer too?

Reply:
Yes, I changed output shape from tensor to parital_shape in range operation to apply this change.
However, since it may be the case that the required output layout of range op is created using cldnn::tensor, in order to prevent it, I constrained output shape by changing output layout to ov::ParitalShape and cldnn::data_types.